WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -656,7 +656,7 @@ private void initPreferences() {
         }
 
         // hide all options except for Delete site and Enable Location if user is not admin
-        if (!mSite.isAdmin()) hideAdminRequiredPreferences();
+        if (!mSite.isWPCom() && !mSite.isSelfHostedAdmin()) hideAdminRequiredPreferences();
```

Comment:
This change means that all wpcom users will have access to the admin preferences and I don't think that's what we want. I think it should be something like this:
`if ((!mSite.isWPCom() && !mSite.isSelfHostedAdmin()) || !mSite.getHasCapabilityManageOptions())`

Also, is the "Enable Location if user is not admin" still a valid comment? I didn't quite understand what that means.